### PR TITLE
gnuplot 5.4.9

### PR DESCRIPTION
**Pull request by p-linnane**
Created by `brew bump`

---

Created with `brew bump-formula-pr`.

### CODE REVIEW

**Review by chenrui333 — APPROVED** [review]

**Comment by p-linnane** [comment]
```
  uic-qt6 -o ui_QtGnuplotSettings.h qtterminal/QtGnuplotSettings.ui
  make[4]: uic-qt6: No such file or directory
  make[4]: *** [ui_QtGnuplotSettings.h] Error 1
```

**Comment by fxcoudert** [comment]
We set `ICU` to the right value, so it shouldn't be overridden. I think this should be filed upstream.

**Comment by p-linnane** [comment]
Opened an upstream issue: https://sourceforge.net/p/gnuplot/bugs/2649/

**Comment by p-linnane** [comment]
@fxcoudert Upstream has provided a configure patch, which has fixed the buld on macOS. Linux is still not able to find `lrelease`. Do you know how we can adjust the arg on Linux to get it to find `lrelease` properly?

File: Formula/g/gnuplot.rb [hunk 1]
```diff
@@ -97,339 +94,27 @@ def install
...
-+          RCC=${QT5LOC}/rcc
-+          LRELEASE=${QT5LOC}/lrelease
-+        fi
-+        CXXFLAGS="$CXXFLAGS -fPIC"
-+        AC_MSG_RESULT([The Qt terminal will use Qt5.])
-+        QTVER="5"
+-      LRELEASE=lrelease-qt6
++      QT6BIN=`$PKG_CONFIG --variable=bindir Qt6Core`
++      if test "x${QT6BIN}" != "x"; then
++          LRELEASE=${QT6LOC}/lrelease
```
Comment (ywwry66) [inline Formula/g/gnuplot.rb:114]:
```suggestion
+          LRELEASE=${QT6BIN}/lrelease
```
This should fix the linux build

Comment (ywwry66) [reply Formula/g/gnuplot.rb:114]: BTW, the upstream patch is actually correct.

Comment (p-linnane) [reply Formula/g/gnuplot.rb]: I must've changed it when testing something. Thanks for pointing that out.

**Review by chenrui333 — APPROVED** [review]

**Pull request merged**